tests/encode/test2_simple/src/main.c

```diff
@@ -239,18 +239,18 @@ void test_pet_raw(void)
 // 			sizeof(serial_rec_input1), &upload);
 // 	zassert_true(ret, "decoding failed.");
 
-// 	zassert_equal(5, upload._Upload_members_count,
+// 	zassert_equal(5, upload.Upload_members_count,
 // 		"expect 5 members");
-// 	zassert_equal(_Member_data, upload._Upload_members[0]
-// 		._Member_choice, "expect data 1st");
-// 	zassert_equal(_Member_image, upload._Upload_members[1]
-// 		._Member_choice, "expect image 2nd");
-// 	zassert_equal(_Member_len, upload._Upload_members[2]
-// 		._Member_choice, "expect len 3rd");
-// 	zassert_equal(_Member_off, upload._Upload_members[3]
-// 		._Member_choice, "expect off 4th");
-// 	zassert_equal(_Member_sha, upload._Upload_members[4]
-// 		._Member_choice, "expect sha 5th");
+// 	zassert_equal(_Member_data, upload.Upload_members[0]
+// 		.Member_choice, "expect data 1st");
+// 	zassert_equal(_Member_image, upload.Upload_members[1]
+// 		.Member_choice, "expect image 2nd");
+// 	zassert_equal(_Member_len, upload.Upload_members[2]
+// 		.Member_choice, "expect len 3rd");
+// 	zassert_equal(_Member_off, upload.Upload_members[3]
+// 		.Member_choice, "expect off 4th");
+// 	zassert_equal(_Member_sha, upload.Upload_members[4]
+// 		.Member_choice, "expect sha 5th");
```

Comment:
IMO we should remove this commented out code. AS in all of it, instead of updating it.